ass/src/main/java/cz/ackee/ass/api/AssRequest.kt

```diff
@@ -1,9 +1,12 @@
 package cz.ackee.ass.api
 
+import com.squareup.moshi.JsonClass
+
 /**
  * Information that is sent for every request, mostly those that can be collected without user's
  * help. User-defined parameters are stored in [customData].
  */
+@JsonClass(generateAdapter = true)
```

Comment:
this will still not work. You need to add kapt dependency to moshi kotlin code gen

Reply:
and please use explicit moshi dependency

Reply:
What do you mean by explicit? There is this moshi dependency

Reply:
there is not moshi dependency, there is depenency on codegen and for retrofit converter, not for moshi itself.